src/components/contact-datalist-item/ContactDatalistItem.tsx

```diff
@@ -15,6 +15,7 @@ export interface ContactDatalistItemProps {
     name: string | null | undefined;
     showAvatar?: boolean;
     subtitle?: React.ReactNode;
+    type?: string;
```

Comment:
Can type ever be null or undefined? If so we might need to treat this similar to how we do the name e.g. type?: string|null|undefined